app/src/main/kotlin/codes/chrishorner/socketweather/home/ForecastUi.kt

```diff
@@ -196,6 +203,47 @@ private fun SectionEntry(@DrawableRes iconRes: Int, iconDesc: String, content: S
   }
 }
 
+@Composable
+private fun Description(
+  description: Description,
+  onToggle: (Boolean) -> Unit
+) {
+  val (text, expandable, expanded) = description
+
+  Column(
+    modifier = Modifier
+      .padding(start = 16.dp, end = 16.dp, bottom = 16.dp)
```

Comment:
I think this might feel a bit nicer if we shift the padding declaration below the `clickable`, letting the touch ripple expand to the screen edges

Reply:
Good call. I've also updated the surrounding padding so that the ripple extends evenly below and above the description. There's no visible change otherwise regardless of the existence of a description.

_Description visible_
<img width="300" src="https://user-images.githubusercontent.com/53217276/135710774-f3432d4a-075a-4415-a1f8-3241145c7d9c.png" /> <img width="300" src="https://user-images.githubusercontent.com/53217276/135710775-201e869a-3ec2-4fd9-b906-55a70cd26919.png" />

_Description hidden_
<img width="300" src="https://user-images.githubusercontent.com/53217276/135710861-28cc7291-a6c7-4af8-8d5a-63e0b3fca8ed.png" /> <img width="300" src="https://user-images.githubusercontent.com/53217276/135710862-09c70e3c-7104-42fe-aa9e-69b918b5fc55.png" />

I'd have enabled view bounds for a better comparison but they don't seem to work for Compose. 🤔

Reply:
Yeah, that's yet to be implemented, but I think it's coming!